app/controllers/videos_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
   def show
     @video = Video.find(params[:id])
+    @video_decorator = VideoDecorator.new(@video)
```

Comment:
There is no need to define a separate instance variable for this. Just do

```ruby
@video = Video.find(params[:id]).decorate
```

That doesn't account for objects that are not found, but you can tweak it to handle that.